### PR TITLE
Feat/tradetrust changes

### PR DESCRIPTION
**Pull request by sumitnitsurat**
- code clean up
- changed text opencert -> tradetrust
- updated image

### CODE REVIEW

**Comment by yehjxraymond** [comment]
image
Still has this, but guess we can remove it in the next clean up

**Comment by yehjxraymond** [comment]
image

Can consider a transparent background

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (yehjxraymond) [inline .github/PULL_REQUEST_TEMPLATE.md]: To be removed. They wont be submitting templates to us.

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "build:static": "next build && next export",
     "start": "next build && next start",
     "start:watch": "nodemon --exec \"npm run start\"",
-    "cname": "echo 'opencerts.io' > out/CNAME",
```
Comment (yehjxraymond) [inline package.json:15]: Can remove. This was previously used for git pages....

**Comment by sumitnitsurat** [comment]
> image
> Still has this, but guess we can remove it in the next clean up
Removed in another PR. this PR majorly targets for text changes and link cleanup.

File: package.json (hunk 2 above)
Comment (sumitnitsurat) [reply package.json:15]: fixed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sumitnitsurat) [reply .github/PULL_REQUEST_TEMPLATE.md]: removed it.

**Comment by sumitnitsurat** [comment]
> image
> 
> Can consider a transparent background

made it transprant

**Review by yehjxraymond — APPROVED** [review]

**Pull request merged**